pageserver/src/tenant.rs

```diff
@@ -2418,6 +2418,7 @@ impl Tenant {
     #[instrument(skip_all, fields(tenant_id=%self.tenant_id))]
     pub async fn gather_size_inputs(
         &self,
+        max_retention_period: Option<u64>,
```

Comment:
Add comment on how this is meant to be used? I think this will be used to try the pitr=0 difference, but unsure.